src/components/DataTable/DataTable.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  DataTable,
+  DataTableBody,
+  DataTableEmpty,
+  DataTableHeader,
+  DataTableRow,
+} from "."
+
+export default {
+  title: "Components/Data Table",
+}
+
+const Template = {
+  children: "Data Table",
+}
+
+export const Empty = () => (
```

Comment:
Exported storybook components names must be globally unique, and `Empty` is not safe to be.

```suggestion
export const DataTableEmptyStory = () => (
)
DataTableEmptyStory. storyName = "Empty"

```